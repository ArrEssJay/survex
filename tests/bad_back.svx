; pos=fail warn=0 error=6
*data normal from to tape compass clino backcompass backclino
*fix 1 0 0 0
1 2 1.0  -  Down -   Up
1 3 1.0  -  Down -  Down
1 4 1.0  -   Up  -  Down
1 5 1.0  -   Up  -   Up
1 A 1.0  -   Up  -   +v
1 B 1.0  -   -v  -  Down
1 C 1.0 000  -89 -   Up
1 D 1.0  -   Down -  30
