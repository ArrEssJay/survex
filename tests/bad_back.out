./bad_back.svx:5:13: error: Plumbed CLINO and BACKCLINO readings can't be in the same direction
./bad_back.svx:7:14: error: Plumbed CLINO and BACKCLINO readings can't be in the same direction
./bad_back.svx:8:14: error: Plumbed CLINO and BACKCLINO readings can't be in the same direction
./bad_back.svx:9:14: error: Plumbed CLINO and BACKCLINO readings can't be in the same direction
./bad_back.svx:10:14: error: CLINO and BACKCLINO readings must be of the same type
./bad_back.svx:11:14: error: CLINO and BACKCLINO readings must be of the same type

Removing trailing traverses...

Concatenating traverses...

Simplifying network...

Calculating network...

Calculating traverses...

Calculating trailing traverses...

Calculating statistics...

Survey contains 3 survey stations, joined by 2 legs.
There are 0 loops.
Total length of survey legs =    2.00m (   2.00m adjusted)
Total plan length of survey legs =    0.00m
Total vertical length of survey legs =    2.00m
Vertical range = 2.00m (from 4 at 1.00m to 2 at -1.00m)
North-South range = 0.00m (from 4 at 0.00m to 4 at 0.00m)
East-West range = 0.00m (from 4 at 0.00m to 4 at 0.00m)
   2 1-nodes.
   1 2-node.
There were 0 warning(s) and 6 error(s) - no output files produced.
